var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -24,6 +25,9 @@ class Zlib(MakefilePackage, Package):
     url = "http://zlib.net/fossils/zlib-1.2.11.tar.gz"
     git = "https://github.com/madler/zlib.git"
 
+    tags = ["core-packages"]
+    libraries = ["libz"]
```

Comment:
```suggestion
    libraries = ["libz", "zlib", "zlibstatic", "zlibd", "zlibstaticd"]
```
This covers all names on all platforms (although I'm not sure how zlib debug libraries are named on MacOS)